packages/datatrak-web/src/Routes.tsx

```diff
@@ -23,10 +39,29 @@ export const Routes = () => {
   return (
     <RouterRoutes>
       <Route path="/" element={<MainPageLayout />}>
-        <Route index element={<LandingPage />} />
-        <Route path="survey" element={<SurveyPageLayout />}>
-          <Route index element={<SurveyPage />} />
-          <Route path="questions" element={<SurveyQuestionsPage />} />
+        <Route
```

Comment:
My understanding is that the user has to be logged in to see any of the pages including the landing page, survey page, admin page etc In which case the PrivateRoute will need to be applied more broadly






[4:21](https://beyondessential.slack.com/archives/D05041PQ30X/p1692850869565859)
In which case the PrivateRoute will need to be applied more broadly